rdt-open-reader/src/main/java/io/ona/rdt/application/RDTApplication.java

```diff
@@ -153,4 +165,23 @@ public ParasiteProfileRepository getParasiteProfileRepository() {
         }
         return parasiteProfileRepository;
     }
+
+    public void verifyUserAuthorization() {
```

Comment:
Do we need this in the Application? Why not have it in utils?

Reply:
for canceling the previous call if it's running, But in Util we can't do that